frontend/src/pages/PropertyEdit.jsx

```
@@ -0,0 +1,772 @@
+                  <textarea
+                    id="description"
+                    name="description"
+                    value={property.description}
+                    onChange={handleInputChange}
+                    required
+                    rows="4"
+                    className={`w-full px-4 py-2 border rounded-md focus:outline-none focus:ring-2 focus:ring-blue-500 ${
+                      hasError('description') ? 'border-red-500' : 'border-gray-300'
+                    }`}
+                    placeholder="Describe your property"
+                  ></textarea>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)